README.md

```diff
@@ -6,14 +6,14 @@ This repo houses the assets used to build the Flux project's landing page at <ht
 
 > **Note**: The sources for some of Flux's documentation are housed in the other Flux repositories within <https://github.com/fluxcd>. Issues and pull requests for this documentation should be registered at that repos.
 >
-> Project       | Docs Site                                        | Github Source
-> ------------- | ------------------------------------------------ | -------------
-> Flux v1       | <https://docs.fluxcd.io>                         | <https://github.com/fluxcd/flux>
-> Helm Operator | <https://docs.fluxcd.io/projects/helm-operator/> | <https://github.com/fluxcd/helm-operator>
-> Flagger       | <https://docs.flagger.app>                       | <https://github.com/fluxcd/flagger>
-> Flux v2       | <https://fluxcd.io/docs>                         | <https://github.com/fluxcd/website>
+> Project       | Docs Site                                 | Github Source
+> ------------- | ------------------------------------------| -------------
+> Flux v1       | <https://fluxcd.io/legacy/flux>           | <https://github.com/fluxcd/website>
+> Helm Operator | <https://fluxcd.io/legacy/helm-operator/> | <https://github.com/fluxcd/website>
+> Flagger       | <https://docs.flagger.app>                | <https://github.com/fluxcd/flagger>
+> Flux v2       | <https://fluxcd.io/docs>                  | <https://github.com/fluxcd/website>
 >
-> We are in the process of moving Flux v1, Helm Operator and Flagger docs into this repository as well.
+> We are in the process of moving Flagger docs into this repository as well.
```

Comment:
We are not in the process of moving Flagger docs, maybe next year, please remove this.